pennylane/transforms/__init__.py

```diff
@@ -45,6 +45,7 @@
 
     ~adjoint
     ~ctrl
+    ~transforms.decompose_single_qubit_unitaries
```

Comment:
Should we include `zyz_decomposition` in the docs here?

Reply:
Sure!

Reply:
We are getting to the size where maybe we should just use `automodapi` rather than hand listing everything?

Reply:
Oh wait, there is an issue, in that some of the functions listed here are top level, and some aren't 😢

Reply:
On second thought, it doesn't really fit in this category since it is itself not a transform - maybe we leave it out for now, but then include it when the compilation transforms get added (it would fit better with those).